index.html

```diff
@@ -196,8 +196,26 @@ <h4>LET'S  CONNECT</h4>
             </ul>
             </div>
            </div>
-              </section> 
+              </section>
+              <section class="contact-area">
+                <div class="contact-me">
+                   <h2 class="contact-title">Contact Me</h2>
+                   <p class="contact-desc">
+                       If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it 
+                   </p>
+                   
+                           <form action="https://formspree.io/forms" method="post">
```

Comment:
Good job, but as the previous reviewer requested, please make sure to use a proper form link. For this visit the [Formspree](https://formspree.io/html) website and create an account. You'll be able to create a form once you log in.

It throws an error currently for the incorrect form link.

![image](https://user-images.githubusercontent.com/34779928/192621261-5230fad7-a6e5-43a8-8a80-0dfcc71629a4.png)